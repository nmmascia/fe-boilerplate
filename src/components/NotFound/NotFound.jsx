import React, { PropTypes } from 'react';

const NotFound = (props) => {
  return (
    <div>{props.text}</div>
  );
};

NotFound.propTypes = {
  text: PropTypes.string,
};

NotFound.defaultProps = {
  text: 'Not found!',
};

export default NotFound;
