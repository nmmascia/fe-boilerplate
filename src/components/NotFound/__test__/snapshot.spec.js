import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from '../NotFound';

test('renders default text', () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders prop text', () => {
  const tree = renderer.create(
    <NotFound
      text="Not Found!!"
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
