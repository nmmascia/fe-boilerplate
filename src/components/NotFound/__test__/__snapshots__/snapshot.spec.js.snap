exports[`test renders default text 1`] = `
<div>
  Not found!
</div>
`;

exports[`test renders prop text 1`] = `
<div>
  Not Found!!
</div>
`;
